pkg/operator/configobservation/auth/auth_serviceaccountissuer.go

```diff
@@ -68,34 +68,38 @@ func observedConfig(
 	}
 
 	if len(existingIssuers) > 0 {
-		existingIssuer = existingIssuers[0]
+		existingActiveIssuer = existingIssuers[0]
 	}
 
-	authConfig, err := getAuthConfig("cluster")
+	operator, err := getOperator("cluster")
 	if apierrors.IsNotFound(err) {
-		klog.Warningf("authentications.config.openshift.io/cluster: not found")
-		// No issuer if the auth config is missing
-		authConfig = &configv1.Authentication{}
+		klog.Warningf("kubeapiserver.operators.openshift.io/cluster: not found")
+		operator = &operatorv1.KubeAPIServer{}
 	} else if err != nil {
 		return existingConfig, append(errs, err)
 	}
 
-	newIssuer = authConfig.Spec.ServiceAccountIssuer
-	if err := checkIssuer(newIssuer); err != nil {
+	newActiveIssuer = getActiveServiceAccountIssuer(operator.Status.ServiceAccountIssuers)
+	if err := checkIssuer(newActiveIssuer); err != nil {
 		return existingConfig, append(errs, err)
 	}
 
-	if len(newIssuer) != 0 {
-		issuerChanged = existingIssuer != newIssuer
+	if len(newActiveIssuer) > 0 {
+		issuerChanged = issuersChanged(
+			existingIssuers,
+			getActiveServiceAccountIssuer(operator.Status.ServiceAccountIssuers),
```

Comment:
nit: this can be replaced with `newActiveIssuer ` and avoids double function call